packages/compass-sidebar/src/components/navigation-items.tsx

```diff
@@ -321,9 +377,10 @@ const mapStateToProps = (
     0
   );
 
-  const isReady = ['ready', 'refreshing'].includes(
-    state.instance?.status ?? ''
-  );
+  const isReady =
+    ['ready', 'refreshing'].includes(state.instance?.status ?? '') &&
+    state.isPerformanceTabSupported !== null;
```

Comment:
Since it is optional wouldn't it be `undefined`, not `null`? Or am I reading this wrong?

Reply:
I'll update the component types in a moment to remove the optional there, but specifically here it's the state, not the component props types, so the value type is as defined in the new store slice: `boolean | null`, where `null` is the value set before we finish running the check